scripts/Reader.py.experimental

```diff
@@ -188,9 +201,11 @@ class Reader(object):
         else:
             with open(path + '/deviceName.txt', 'r') as f:
                 device_name = f.read().rstrip().split(';', 1)[0]
+            with open(path + '/../settings/Mode_Legacy', 'r') as f:
+                mode_legacy = f.read().rstrip().split(';', 1)[0] == 'ON'
```

Comment:
I think it should be the other way around, making legacy really the default.

So it should be "MODE_FULLID" with Value "ON".
If this file isn't present or the value isn't matching, the legacy mode is used.
So only an active change of the user will result in using the new mode.

Reply:
It was my intent to make legacy the default. With the added error handling on missing `Mode_Legacy` it know really is. On new installations or if `global.conf` is recreated the `Mode_Legacy` file will be created triggered by the start-up script.

Reply:
Just one case left:
If the content of the file does not match (anything other then `ON`) the legacy mode is not used.

To really have it bulletproof, the "new" mode needs to be checked and all other cases result in legacy mode.
Thats what the suggestions is about :)

Reply:
I see your point. Followed your suggestion, cf. d618ec0db13a052664effedcf585677624a13bb5.